olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -893,29 +893,29 @@ spec:
                       - name: "IMAGE_net-istio-webhook__webhook"
                         value: "registry.ci.openshift.org/openshift/net-istio-webhook:knative-v1.11"
                       - name: "IMAGE_eventing-controller__eventing-controller"
-                        value: "registry.ci.openshift.org/openshift/knative-eventing-controller:knative-v1.11"
+                        value: "registry.ci.openshift.org/openshift/knative-eventing-controller:knative-v1.13"
```

Comment:
that should be 1.12, right?

Reply:
We are planning to go directly to 1.13 to have OIDC and eventtype discovery in

Reply:
🤷  this could cause interesting problems. So operator will be 1.12, go deps (in S-O) will be 1.12 and actual running images 1.13? 

/cc @skonto @dsimansk

Reply:
@creydr I think in the past you did skip version but only at the content level not real version is that true?

Reply:
> I think in the past you did skip version but only at the content level not real version is that true?

Let me bring in @pierDipi and @matzew as I don't know the details how this was done the last time

Reply:
> @creydr I think in the past you did skip version but only at the content level not real version is that true? Would not this confuse the customers as they would expect 1.12 and the usual Knative version upgrade policy?

Yep, that's what Matthias already mentioned. IMO the content was 1.13, but appeared as 1.12 on the surface level, i.e. image versions.

Reply:
For the manifests, we have done that before. Shipping a newer version as `-1`

Reply:
This is a test how this really goes. the operand is in theory indepdent from the operator (and APIs are stable)